safe_video/number_plate_recognition/utils.py

```diff
@@ -37,6 +39,20 @@ def merge_results(result1: Results, result2: Results) -> Results:
     merged_result.boxes.data = merged_data
     return merged_result
 
+def merge_results_list(results: list[Results]) -> Results:
+    """
+    Merges the bounding boxes of multiple YOLO results and also updates the class mapping.
+
+    Args:
+        results (list[Results]): List of YOLO results
+
+    Returns:
+        Results: Merged YOLO result containing all bounding boxes from all results and also updated class mapping
+    """
+    merged_result: Results = None
+    for result in results: merged_result = merge_results(merged_result, result)
+    return merged_result
+
 def find_key_by_value(dictionary: dict, value: str) -> int:
     return list(dictionary.keys())[list(dictionary.values()).index(value)]
 
```

Comment:
I could not mark it in git but change `filter_results` to `filter_detection_results_by_class` so it directly clear what is happening

Reply:
I think the filter function should be extendend with additional parameters making the general name fitting https://github.com/Cari1111/NumberPlateRecognition/issues/49